README.md

```diff
@@ -1 +1,182 @@
-# Budget-App
\ No newline at end of file
+
+<div align="center">
+  <br/>
+
+  <h3><b>Budget Track</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Budget Track] <a name="about-project"></a>
+
+**[Budget Track]** The Ruby on Rails Budget App is centered around the creation of a mobile web application designed for budget management. In this application, users can efficiently oversee their financial transactions by categorizing them. By doing so, they can gain insight into their spending patterns, allowing them to track their expenses and understand where their money is allocated. This project not only showcases the use of Ruby on Rails but also serves as a practical tool for individuals to maintain control over their finances.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby on Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li>RSPEC</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://cizzla.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Presentation Video <a name="presentation-video"></a>
+
+- [Presentation Video Link](https://www.loom.com/share/19778c6fe59f495ab016b8209215bd8f?sid=720b438c-56be-4546-982d-7979876f19fb)
```

Comment:
- [ ] Good job presenting the project it looks very professional, however, the requirements say you should also present a piece of code you're proud of.